RELEASE_NOTES.rst

```diff
@@ -1,3 +1,64 @@
+coala-bears 0.11.0
+==================
+
+This release has been brought to you by 30 contributor contributing over 101
+commits.
+
+**Bug Fixes**
```

Comment:
please move this section to the bottom to be consistent with the previous relnotes